Firmware/Configuration.h

```diff
@@ -22,6 +22,7 @@
 #define EEPROM_FILAMENTUSED 4081
 // uint32_t
 #define EEPROM_TOTALTIME 4077
+#define EEPROM_FILAMENT_NOTIFICATION 4076
```

Comment:
I think should not use that address 4076 because it is used by the line beneath !!!